### PR TITLE
feat: add introduce application template

### LINKED ISSUE #210: Issue with Porter - need checkbox for inclusion in Application Portfolio Management (APM)
### Describe the problem you are having
There is no good way to update our APM with porter the way it is now.  

#### Should a new checkbox be added

Values
publish to APM? 
I am unsure if that is enough to then add it to the APM in ServiceNow

### LINKED ISSUE #212: Add New Application Template
We have a template for [deprecating an app](https://github.com/agrc/porter/blob/main/.github/ISSUE_TEMPLATE/3-deprecate-application.md) but not for introducing one. Is there any consensus on creating one? It would help us remember things like #210. Maybe there are other steps that we should be verifying when we create new apps?

### PR DESCRIPTION
**Pull request by stdavis**
Closes #212
Closes #210

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/3-introduce-application.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+_A short summary of the situation._
+
+## Action items
+
+1. _Assign a person who should complete the task by replacing `name` with their github `@name`._
+1. _Check [x] the box when the task is completed and add the date of completion._
+1. _~Strike~ out all items that do not apply._
+
+- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
+- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (name, completed: `2022/00/00`)
```
[the remark this answers is not in the document]
Comment (stdavis) [reply .github/ISSUE_TEMPLATE/3-introduce-application.md:25]: I added a step for archer diagrams above this one to help with this.

**Comment by stdavis** [comment]
@nathankota I see you as the primary user for this issue template so I want it to match what you want. I added all of your suggestions. Feel free to make any changes (or I'm happy to make them for you), or approve this PR.

File: .github/ISSUE_TEMPLATE/3-introduce-application.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] SOW or similar agreement (name, completed: `2022/00/00`)
+- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
+- [ ] Create Archer security diagram (name, completed: `2022/00/00`)
+- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (name, completed: `2022/00/00`)
+- [ ] Create maintenance contract (name, completed: `2022/00/00`)
+- [ ] Maintenance contract (name, completed: `2022/00/00`)
+- [ ] Service Now task (name, completed: `2022/00/00`)
+- [ ] GitHub repo (name, completed: `2022/00/00`)
+  - Repo URL:
+- [ ] Primary client contact (name, completed: `2022/00/00`)
```
Comment (steveoh) [inline .github/ISSUE_TEMPLATE/3-introduce-application.md:33]:
```suggestion
- [ ] SOW or similar agreement (@nathankota, completed: `2022/00/00`)
- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
- [ ] Create Archer security diagram (@nathankota, completed: `2022/00/00`)
- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (@nathankota, completed: `2022/00/00`)
- [ ] Create maintenance contract (@nathankota, completed: `2022/00/00`)
- [ ] Maintenance contract (@nathankota, completed: `2022/00/00`)
- [ ] Service Now task (@nathankota, completed: `2022/00/00`)
- [ ] GitHub repo (name, completed: `2022/00/00`)
  - Repo URL:
- [ ] Primary client contact (@nathankota, completed: `2022/00/00`)
```

File: .github/ISSUE_TEMPLATE/3-introduce-application.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+1. _Assign a person who should complete the task by replacing `name` with their github `@name`._
+1. _Check [x] the box when the task is completed and add the date of completion._
+1. _~Strike~ out all items that do not apply._
+
+- [ ] SOW or similar agreement (name, completed: `2022/00/00`)
+- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
+- [ ] Create Archer security diagram (name, completed: `2022/00/00`)
+- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (name, completed: `2022/00/00`)
+- [ ] Create maintenance contract (name, completed: `2022/00/00`)
+- [ ] Maintenance contract (name, completed: `2022/00/00`)
```
Comment (steveoh) [inline .github/ISSUE_TEMPLATE/3-introduce-application.md:29]: **quibble**: these two seem like duplicates

File: .github/ISSUE_TEMPLATE/3-introduce-application.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+name: Adding an application
+about: Application Checklist
+title: Introduce <application>
+labels: "introduction, porter"
+assignees: "@steveoh, @gregbunce, @rkelson"
```
Comment (steveoh) [inline .github/ISSUE_TEMPLATE/3-introduce-application.md:6]: **suggestion**: let nathan run with this one

```suggestion
assignees: "@nathankota"
```

File: .github/ISSUE_TEMPLATE/3-introduce-application.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+- [ ] SOW or similar agreement (@nathankota, completed: `2022/00/00`)
+- [ ] Add to [gis.utah.gov](https://gis.utah.gov/developer/application) (name, completed: `2022/00/00`)
+- [ ] Create Archer security diagram (@nathankota, completed: `2022/00/00`)
+- [ ] Add to Application Porfolio Management ("Business Applications" in ServiceNow) (@nathankota, completed: `2022/00/00`)
+- [ ] Create maintenance contract (@nathankota, completed: `2022/00/00`)
+- [ ] Service Now task (@nathankota, completed: `2022/00/00`)
+- [ ] GitHub repo (name, completed: `2022/00/00`)
+  - Repo URL:
+- [ ] Primary client contact (@nathankota, completed: `2022/00/00`)
```
Comment (steveoh) [inline .github/ISSUE_TEMPLATE/3-introduce-application.md:32]:
```suggestion
- [ ] Create Archer security diagram (@nathankota, completed: `2022/00/00`)
  - Diagram URL: 
- [ ] Add to Application Portfolio Management ("Business Applications" in ServiceNow) (@nathankota, completed: `2022/00/00`)
  - Id: 
- [ ] Create maintenance contract (@nathankota, completed: `2022/00/00`)
- [ ] Service Now task (@nathankota, completed: `2022/00/00`)
  - Name: 
- [ ] GitHub repo (name, completed: `2022/00/00`)
  - Repo URL: 
- [ ] Primary client contact (@nathankota, completed: `2022/00/00`)
  - Name: 
```

Comment (stdavis) [reply .github/ISSUE_TEMPLATE/3-introduce-application.md]: I applied this on my own since the suggestion became outdated.

**Pull request merged**